### PR TITLE
Use latest rag content

### PR DESCRIPTION
**Pull request by asamal4**
## Description

Point to latest rag content (last changed through https://github.com/openshift/lightspeed-rag-content/pull/28)

### CODE REVIEW

**Review by tisnik — COMMENTED** [review]
it looks ok

**Comment by tisnik** [comment]
@asamal4 what about to create PR just with the fix? (remove command)

**Comment by bparees** [comment]
@xrajesh this is another thing to be aware of.....any time we bump the SHA that OLS is pointing to for RAG content, we need to first tag that SHA in quay before merging the PR, otherwise the SHA could get garbage collected and then our OLS builds would start failing.

https://quay.io/repository/openshift/lightspeed-rag-content?tab=tags

i've now tagged `daaaf9e8e610` so this PR is ok to merge.

@asamal4 @syedriko remove the hold whenever you're ready

/approve
/lgtm

File: Containerfile [hunk 1]
```diff
@@ -1,5 +1,5 @@
 # vim: set filetype=dockerfile
-ARG LIGHTSPEED_RAG_CONTENT_DIGEST=sha256:8dd07db7cfe7cba0dcbfe8eb3c53a6f9bbca324bbd98cd78fa8ffe05294c5e84
+ARG LIGHTSPEED_RAG_CONTENT_DIGEST=sha256:daaaf9e8e610f6cf33bcb714f1198a3c027b9a18b265853bc73b4142866deda0
```
Comment (bparees) [inline Containerfile:2]: @syedriko @asamal4 let's add a comment to the dockerfile above this line to tell/remind people that the SHA value being referenced must be tagged in quay prior to merging the change.

**Comment by xrajesh** [comment]
> @xrajesh this is another thing to be aware of.....any time we bump the SHA that OLS is pointing to for RAG content, we need to first tag that SHA in quay before merging the PR, otherwise the SHA could get garbage collected and then our OLS builds would start failing.
> 
> https://quay.io/repository/openshift/lightspeed-rag-content?tab=tags
> 
> i've now tagged `daaaf9e8e610` so this PR is ok to merge.
> 
> @asamal4 @syedriko remove the hold whenever you're ready
> 
> /approve /lgtm

Sure Ben - Noted .  Thank you .

**Comment by xrajesh** [comment]
> @xrajesh this is another thing to be aware of.....any time we bump the SHA that OLS is pointing to for RAG content, we need to first tag that SHA in quay before merging the PR, otherwise the SHA could get garbage collected and then our OLS builds would start failing.
> 
> https://quay.io/repository/openshift/lightspeed-rag-content?tab=tags
> 
> i've now tagged `daaaf9e8e610` so this PR is ok to merge.
> 
> @asamal4 @syedriko remove the hold whenever you're ready
> 
> /approve /lgtm

@bparees We should TAG it here in openshift-lightspeed https://quay.io/repository/openshift-lightspeed/lightspeed-rag-content?tab=tags not https://quay.io/repository/openshift/lightspeed-rag-content?tab=tags

**Comment by bparees** [comment]
> @bparees We should TAG it here in openshift-lightspeed https://quay.io/repository/openshift-lightspeed/lightspeed-rag-content?tab=tags not https://quay.io/repository/openshift/lightspeed-rag-content?tab=tags

ugh, good catch.  tagged in the correct place now:
https://quay.io/repository/openshift-lightspeed/lightspeed-rag-content?tab=tags

**Pull request merged**